package.json

```
@@ -5,6 +5,9 @@
   "description": "k6 Studio",
   "repository": "github:grafana/k6-studio",
   "main": ".vite/build/main.js",
+  "engines": {
+    "node": ">=20.0.0 <=23.0.0"
```

This config would still allow using exact v23.0.0, which may not be the best, because it won't have any minor or patch updates.

Another question, if we actually need to support this wide range of versions, could we lock it at current LTS (v22) instead? This would also involve updating .nvmrc and git workflows to use v22.

```suggestion
    "node": "^22.0.0"
```

---

That's a great suggestion! I just changed it to v22.0.0 in workflow files and package.json.